Makefile

```diff
@@ -1,60 +1,67 @@
-VENV = venv
-PYTHON = $(VENV)/bin/python3
-PIP = $(VENV)/bin/pip
+PYTHON = python3
 
+VENV      = venv
+ACTIVATE := source $(VENV)/bin/activate
 
-$(VENV)/bin/activate: requirements.txt
-	python3 -m venv $(VENV)
-	$(PIP) install --upgrade pip
-	$(PIP) install -r requirements.txt
+DOCKER = docker
+TESTDB = solver-rewards-test-db
```

Comment:
Prevent issues with Docker image and container main collisions, as well as allow things to work with other `docker` alternatives such as Podman:
```
make db DOCKER=podman TESTDB=my-weird-image-name
```